### PR TITLE
Adding support for Ukrainian + Correcting some mistakes

### LINKED ISSUE #142: [Feat] - Add Ukrainien - [i18n]

### PR DESCRIPTION
**Pull request by bogdzn**
| Q             | A
| ------------- | ---
| Branch?       | 
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | yes
| Tickets       | Fix #142 

I made a lot of commits there, maybe you should use 'Squash and Merge' instead of simple merge

Please let me know if i can be of any utility to make this a better PR.

### CODE REVIEW

File: client/src/assets/i18n/ua.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      "SELECTVULNSTYPE": "Вид вразливості",
+      "SELECTIMPACTS": "Вплив"
+    },
+    "vulns-edit": {
+      "NAME": "ім'я",
+      "DESCRIPTION": "Опис",
+      "REMEDIATION": "Санація"
+    }
+  }
+
```
Comment (darkweak) [inline client/src/assets/i18n/ua.json:113]: Add new line at the end of the file.

**Review by darkweak — CHANGES_REQUESTED** [review]

File: client/src/assets/i18n/ua.json (hunk 1 above)
Comment (bogdzn) [reply client/src/assets/i18n/ua.json]: Done, fixed wrong indentation as well

File: client/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -74,11 +74,11 @@
     "ENDAT": "Date de fin",
     "GENERATE-REPORT": "Génération du rapport",
     "DUMPJSON": "Téléchargement des données mission (JSON)",
-    "CODI": "Ajoutez ou modifier le lien codiMD",
+    "CODI": "Ajouter ou modifier un lien codiMD",
```
Comment (darkweak) [inline client/src/assets/i18n/fr.json:77]: Actually there can be only one codimd link
```suggestion
    "CODI": "Ajouter ou modifier le lien codiMD",
```

File: client/src/assets/i18n/fr.json [hunk 3]
```diff
@@ -74,11 +74,11 @@
...
     "BURP": "Téléchargement du fichier de configuration BurpSuite",
     "CLIENTS": "Client",
-    "IMPORT": "Import de plusieurs domaine",
-    "ADDNEWHOST": "Ajoutez un host manuellement",
+    "IMPORT": "Import de plusieurs domaines",
```
Comment (darkweak) [inline client/src/assets/i18n/fr.json:80]:
```suggestion
    "IMPORT": "Import de plusieurs noms de domaine",
```

**Review by darkweak — CHANGES_REQUESTED** [review]

File: client/src/assets/i18n/fr.json (hunk 2 above)
Comment (bogdzn) [reply client/src/assets/i18n/fr.json]: done

File: client/src/assets/i18n/fr.json (hunk 3 above)
Comment (bogdzn) [reply client/src/assets/i18n/fr.json]: done

**Review by darkweak — APPROVED** [review]

**Pull request merged**